spec/models/support_interface/providers_filter_spec.rb

```diff
@@ -85,6 +85,21 @@
       expect(filter.filter_records(Provider.all)).to match_array [provider_with_provider_user, provider_without_provider_user]
     end
 
+    it 'filters by accredited provider' do
+      accredited_provider = create(:provider, name: 'Accredited Provider')
+      training_provider1 = create(:provider)
+      training_provider2 = create(:provider)
```

Comment:
Given that we are passing records into the `#filter_records` method would it be worth using FactoryBot `build` in order to optimise test speed by not writing to the db?

Reply:
I guess same goes for the other filter specs

Reply:
The methods query the db though right?